tasks/fix-cat3.yml

```diff
@@ -89,27 +89,54 @@
       - mount
       - tmp
 
-- name: "LOW | RHEL-07-021600 | PATCH | The file integrity tool must be configured to verify Access Control Lists (ACLs)."
-  command: "true"
-  changed_when: no
+- block:
+    - name: "LOW | RHEL-07-021600 | PATCH | The file integrity tool must be configured to verify Access Control Lists (ACLs)."
```

Comment:
We should clarify the code guidelines for indentation.  I see that @shepdelacreme has used two additional spaces before the '-' inside a block, and also 4 additional spaces, all in different parts of the same commit.  my reading of the code guidelines (which says 4 spaces) suggests 4 additional spaces, which is what we do for "tags", "when", etc.

Reply:
Will see what it takes to get emacs to make this more automatic. Or will just do it the manual way.

Reply:
[yamllint](https://github.com/adrienverge/yamllint), perhaps? No idea if it's comprehensive enough for everything, but it may handle many of the style fixes.

Reply:
Sorry for the confusion with indentation here. The original style guide stated 4 spaces. However nothing has automated the checking/enforcement of that so its not consistent everywhere. For now strive for 4 spaces because its in the style guide and I think it is more readable. I will work on getting `yamllint` or `ansible-lint` setup on this repo and see if we can enforce it as part of the CI checks.